### ISSUE TITLE
Warnings (?) caused by Serialization / Polymake

### ISSUE THREAD

**Issue by fingolfin** [oscar-system/Oscar.jl#1083]
Running the test suite, I see this:
```
...
Test Summary: | Pass  Total
Serialization |   23     23
	(in cleanup) can't create a new copy of /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/moae.sop: No such file or directory
polymake:  WARNING: could not rename temporary file /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/poly36163Taaaa0011 to destination /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/moae.sop: No such file or directory
	(in cleanup) can't create a new copy of /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/moae.sop: No such file or directory
polymake:  WARNING: could not rename temporary file /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/poly36163Taaaa0012 to destination /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/moae.sop: No such file or directory
	(in cleanup) can't create a new copy of /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/lp.poly: No such file or directory
polymake:  WARNING: could not rename temporary file /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/poly36163Taaaa0013 to destination /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/lp.poly: No such file or directory
	(in cleanup) can't create a new copy of /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/lp.poly: No such file or directory
polymake:  WARNING: could not rename temporary file /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/poly36163Taaaa0014 to destination /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/lp.poly: No such file or directory
	(in cleanup) can't create a new copy of /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/nfsquare.fan: No such file or directory
polymake:  WARNING: could not rename temporary file /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/poly36163Taaaa0015 to destination /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/nfsquare.fan: No such file or directory
	(in cleanup) can't create a new copy of /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/nfsquare.fan: No such file or directory
polymake:  WARNING: could not rename temporary file /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/poly36163Taaaa0016 to destination /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/nfsquare.fan: No such file or directory
	(in cleanup) can't create a new copy of /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/square.poly: No such file or directory
polymake:  WARNING: could not rename temporary file /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/poly36163Taaaa0017 to destination /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/square.poly: No such file or directory
	(in cleanup) can't create a new copy of /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/square.poly: No such file or directory
polymake:  WARNING: could not rename temporary file /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/poly36163Taaaa0018 to destination /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/square.poly: No such file or directory
	(in cleanup) can't create a new copy of /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/posorth.cone: No such file or directory
polymake:  WARNING: could not rename temporary file /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/poly36163Taaaa0019 to destination /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/posorth.cone: No such file or directory
	(in cleanup) can't create a new copy of /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/posorth.cone: No such file or directory
polymake:  WARNING: could not rename temporary file /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/poly36163Taaaa0020 to destination /private/var/folders/9c/w7mlr03x2yd67g7n6384lvlc0000gp/T/jl_El4HRp/posorth.cone: No such file or directory
#I  Forcing finiteness test
Test Summary:     | Pass  Total
SimplicialComplex |   26     26
...
```

CC @lkastner @benlorenz

**Comment by benlorenz** [comment]
These warnings are harmless and happen when polymake tries to store changes in a saved file but julia has already removed the corresponding temporary directory. This should not happen during normal usage.

I think the reason is that julia is rather quick to clean up the tempdir but the objects created in that do block seem to live a lot longer which polymake is not really pepared for. I will try if I can find a reasonable way to avoid this.

**Comment by benlorenz** [comment]
It seems that this is solved by Lars in #1000 anyway due to the reorganization, I can't find it in the logs from that PR anymore.

**Comment by lkastner** [comment]
> It seems that this is solved by Lars in #1000 anyway due to the reorganization, I can't find it in the logs from that PR anymore.

Yes, this is because Julia is writing and reading files there and polymake only does parsing for its dedicated parts.
